lib/api/sibling-nav.js

```diff
@@ -4,22 +4,23 @@ import { queryAPI } from "@/lib/fetch";
 export async function addSiblings(entryData, site = "default") {
   if (!entryData) return null;
 
-  const { uri, level } = entryData;
-
-  const siblings = await getSiblings(uri, level, site);
-
+  const { uri, level, parent } = entryData;
+  const parentId = parent?.id;
+  const siblings = parentId
+    ? await getSiblings(parentId, uri, level, site)
+    : {};
```

Comment:
If there isn't a parentId then don't attempt to grab the siblings

Reply:
I don't see any issue but I think `null` could be a better alternative than `{}`

The `SiblingNavigation` component expects a `siblings` object with `next, prev` properties OR it returns `null` if `siblings` is a falsey value. It probably doesn't really matter since afterwards it checks for the existence of `next,prev` anyway, and the parent needs to have children for the sibling nav to show anyway.